src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -209,17 +224,29 @@ protected int doWork() {
             baseQHisto.increment(i, baseQHistogramArray[i]);
         }
 
-        final WgsMetrics metrics = generateWgsMetrics();
-        metrics.GENOME_TERRITORY = (long) histo.getSumOfValues();
-        metrics.MEAN_COVERAGE = histo.getMean();
-        metrics.SD_COVERAGE = histo.getStandardDeviation();
-        metrics.MEDIAN_COVERAGE = histo.getMedian();
-        metrics.MAD_COVERAGE = histo.getMedianAbsoluteDeviation();
+        // Construct and write the outputs
+        final Histogram<Integer> baseQHetHisto = new Histogram<Integer>("value", "baseq_count");
+        for (int i=0; i<baseQHetSensHistogram.length; ++i) {
+            if(i>0 && i<17) {
+                baseQHetHisto.increment(0, baseQHetSensHistogram[i]);
+                baseQHetHisto.increment(i, 0);
```

Comment:
It does, without incrementing that bin, that bin doesn't exist in the histogram. When a bin is initialized it is set at 0 but they aren't auto filled at set up. 

If this way of initializing empty bins doesn't seem best, there is the option of prefilling all the bins in the histogram with 0 before we increment them in the loop?